### PR TITLE
On Android feeds do not load next page when list is scrolled to the bottom #249

### PR DESCRIPTION
**Pull request by antonm76**
1. onEndReachedThreshold has to be >0 to onEndReached to be fired on Android
2. Footer with activity indicator is added

### CODE REVIEW

File: mobile/src/feeds/home-feeds-screen.tsx [hunk 1]
```diff
@@ -47,60 +50,82 @@ const mapDispatchToProps = (dispatch: Dispatch<any>): FeedScreenDispatchProps =>
...
+                onEndReachedThreshold={0.2}
+                refreshing={false}
+                onRefresh={this.onRefresh}
+                ListFooterComponent={this.footer}
+            /> :
+            <LoadingView/>;
     }
 
+    //----------------------------------------------------------------------------
+    // noinspection JSMethodCanBeStatic
```
Comment (DesmanLead) [inline mobile/src/feeds/home-feeds-screen.tsx]: Why not to actually make it static?

Comment (antonm76) [reply mobile/src/feeds/home-feeds-screen.tsx]: I would rather make them lambdas

Comment (antonm76) [reply mobile/src/feeds/home-feeds-screen.tsx]: done: 9ade2597c215d48be39585e79cec5fdc7193608e

**Comment by antonm76** [comment]
Done

**Pull request merged**